ocrd/processor/base.py

```diff
@@ -2,8 +2,8 @@
 import json
 import subprocess
 from deprecated.sphinx import deprecated
-from ocrd.utils import getLogger
-from ocrd.validator import ParameterValidator
+from ocrd_shared.utils import getLogger
+from ocrd_models.validator import ParameterValidator
 
```

Comment:
Dependency is

```
ocrd_shared
  ocrd_models
    ocrd
```

So, if you want to NOT have to import the parameter validation routines when using ocrd_models, moving them to ocrd_shared won't help since they are transitively required.

On the other hand, the only requirement that comes with it is json_schema which is a less heavy requirement than e.g. lxml.